opencti-platform/opencti-front/src/private/components/data/connectors/Connector.jsx

```diff
@@ -567,6 +859,42 @@ const Connector = createRefetchContainer(
         connector_type
         connector_scope
         connector_state
+        connector_user_id
+        connector_user {
```

Comment:
I'm a little worried that we might query connector_user with roles & groups too many times since this component refetches the connector every 10 seconds

Reply:
This raises a number of questions !